app/views/profiles/show.html.erb

```diff
@@ -13,31 +13,30 @@
         </div>
         <div id='friend-request-button' class='absolute -top-4 -right-2 '>
           <span class='bg-blue-200 py-1 px-2 rounded-full opacity-50'>Request Friend</span>
-          <% if current_user == @profile.user %>
-            <%= link_to 'Edit', edit_user_profile_path(@user), id:'edit-profile-button', class:'bg-blue-200 py-2 px-2 rounded-full w-9', data: {turbo_frame: 'edit-profile-modal'}%>
+          <% if current_user == @user %>
+            <%= link_to 'Edit', edit_profile_path(@user), id:'edit-profile-button', class:'bg-blue-200 py-2 px-2 rounded-full w-9', data: {turbo_frame: 'edit-profile-modal'}%>
           <% end %>
         </div>
       </div>
       <div id='about-body' class='relative pt-14'>
-        <div id='name' class='font-bold font-sans'><%= @profile.user.display_name%> - <%=@profile.slug%></div>
```

Comment:
You don't like the slug?

Reply:
I honestly can't tell the diff between the profile slug and the user unique_name... it seems like they serve the same function?

Reply:
The idea of the slug was to be a cute little tag line, like when previewing a user  in a friendship view, as opposed to the description which could be a paragraph. Not really necessary, just threw it on there.